server/lib/nl/detection/variable.py

```diff
@@ -60,8 +57,11 @@ def detect_vars(orig_query: str,
   # the potential areas for improvement. For now, this removal blanket removes
   # any words in ALL_STOP_WORDS which includes contained_in places and their
   # plurals and any other query attribution/classification trigger words.
-  query_monovar = shared_utils.remove_stop_words(orig_query,
-                                                 query_util.ALL_STOP_WORDS)
+  if dargs.include_stop_words:
```

Comment:
slightly confused, is this argument supposed to strip stop words or include stop words? because seems like it's stripping stop words here?

Reply:
Thanks, yes, when I renamed it per Bo's suggestion, I messed it up.

Fixed now, updated screenshot with new param.